cybereason_poller.py

```diff
@@ -32,35 +40,49 @@ def do_poll(self, connector, param):
+                MALWARE_HISTORICAL_DAYS_KEY
+            )
             if phantom.is_fail(ret_val):
                 return action_result.get_status()
 
             if is_first_poll:
-                connector.save_progress("This is a first time poll. We will poll for malops from the last {days} days", days=malop_historical_days)
+                connector.save_progress(
+                    "This is a first time poll. We will poll for malops from the last {days} days", days=malop_historical_days
+                )
                 malop_start_time = current_time + datetime.timedelta(days=-malop_historical_days)
-                connector.save_progress("This is a first time poll. We will poll for malware from the last {days} days", days=malware_historical_days)
+                connector.save_progress(
+                    "This is a first time poll. We will poll for malware from the last {days} days", days=malware_historical_days
+                )
                 malware_millisec_since_last_poll = malware_historical_days * 60 * 60 * 24 * 1000
                 state["is_first_poll"] = False
             else:
                 last_poll_timestamp = datetime.datetime.fromtimestamp(state["last_poll_timestamp"])
                 malop_start_time = last_poll_timestamp
                 malware_millisec_since_last_poll = round((current_time - last_poll_timestamp).total_seconds() * 1000)
             state["last_poll_timestamp"] = current_time.timestamp()
-            connector.save_progress("Getting malops between {start_time} and {current_time}", start_time=malop_start_time, current_time=current_time)
+            connector.save_progress(
+                "Getting malops between {start_time} and {current_time}", start_time=malop_start_time, current_time=current_time
+            )
             connector.save_progress("Getting malware for the last {msec} milliseconds", msec=malware_millisec_since_last_poll)
 
             # Initialize the session that will be used throughout the poller
             self.cr_session = CybereasonSession(connector).get_session()
             malop_start_time_microsec_timestamp = round(malop_start_time.timestamp() * 1000)
             # When called as a scheduled poll, max_container count comes as 4294967295 which causes a Cybereason API error.
             container_count = min(int(param.get(phantom.APP_JSON_CONTAINER_COUNT)), 5000)
-            success = success & self._fetch_and_ingest_malops(connector, config, malop_start_time_microsec_timestamp, container_count)
-            success = success & self._fetch_and_ingest_malwares(connector, config, malware_millisec_since_last_poll, container_count)
+            success = success & self._fetch_and_ingest_malops(connector, config,
+                    malop_start_time_microsec_timestamp, container_count)
+            success = success & self._fetch_and_ingest_malwares(connector, config,
+                    malware_millisec_since_last_poll, container_count)
```

Comment:
nit: If it's not exceeded the line length, Should we move all params to the next line? 
```python
 success = success & self._fetch_and_ingest_malops(
                  connector, config, malop_start_time_microsec_timestamp, container_count)
```

Reply:
Updated